src/audio/igo_nr/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+target_include_directories(sof_public_headers INTERFACE ${CMAKE_CURRENT_SOURCE_DIR})
+
+add_local_sources(sof igo_nr.c)
+
+cmake_policy(SET CMP0079 NEW)
```

Comment:
why is this needed?

Reply:
These code, which were used to get rid of error during linking to the static library, will be removed. Because I just notice that SoF framework provides `sof_add_static_library` which is much more elegant.